src/foam/u2/wizard/wizardlet/SuccessWizardletView.js

```diff
@@ -15,13 +15,12 @@ foam.CLASS({
   css: `
     ^ {
       text-align: center;
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-      align-items: center;
       margin: 8rem 0;
-      height: 100%;
-      gap: 42px;
+    }
+    ^centered {
+      align-self: center;
```

Comment:
How does this work if the parent Div isnt flex anymore?

Reply:
i forgot to remove this